core/state_processor.go

```diff
@@ -320,7 +320,7 @@ func (kvm *keyValueMigrator) prepare() {
 				var currAddr common.Address
 				var currPoint *verkle.Point
 				for i := range batch {
-					if batch[i].branchKey.addr != currAddr {
+					if batch[i].branchKey.addr != currAddr || currAddr == (common.Address{}) {
```

Comment:
this is to force the initialization of `currPoint` so that it doesn't panic in `GetTreeKeyWithEvaluatedAddress`.

Reply:
If `currAddr == (common.Address{})` then from my perspective `batch[i].branchKey.addr != currAddr` should be true.